sql/moz-fx-data-shared-prod/telemetry_dev_cycle_external/telemetry_probes_stats_v1/query.py

```diff
@@ -0,0 +1,109 @@
+"""Telemetry probes data - download from API, clean and upload to BigQuery."""
+
+import logging
+
+import click
+import requests
+from google.cloud import bigquery
+
+API_BASE_URL = "https://probeinfo.telemetry.mozilla.org"
+SCHEMA = [
```

Comment:
Same as this comment in one of the other PRs:

https://github.com/mozilla/bigquery-etl/pull/4723/files#r1432986857

Should we try and see if we could just load this from the schema.yaml to avoid having to maintain the schema in multiple places?

Reply:
uhhh interesting ... but it feels a bit of a henn egg thing? because the schema.yaml can be updated bqetl and might not be the one that is manully written down?
so this feels safer to me? but i'll think about it